### PR TITLE
Integrate maintenance fork into phpenv

### PR DESCRIPTION
**Pull request by madumlao**
Long overdue resolution to #57 
and #83

### CODE REVIEW

File: libexec/phpenv-configure [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# Summary: Edit the current PHP's php.ini configuration file at location
+
+set -e
+[ -n "$PHPENV_DEBUG" ] && set -x
+
+vi "$(php --ini|grep Loaded|awk '{print $NF}')"
```
Comment (glensc) [inline libexec/phpenv-configure]: rather `${EDITOR:-vi}`

File: README.md [hunk 2]
```diff
@@ -23,19 +40,21 @@ directory for the selected version at the beginning of your `$PATH`
...
+If you prefer a manual approach, follow the steps below.
 
 This will get you going with the latest version of phpenv and make it
 easy to fork and contribute any changes back upstream.
 
 1. Check out phpenv into `~/.phpenv`.
 
-        $ cd
-        $ git clone git://github.com/humanshell/phpenv.git .phpenv
+        $ git clone git://github.com/madumlao/phpenv.git ~/.phpenv
```
[the remark this answers is not in the document]
Comment (madumlao) [reply README.md]: Integrated into #86

File: libexec/phpenv-configure (hunk 1 above)
Comment (madumlao) [reply libexec/phpenv-configure]: I'll look into this. Not sure which is more "standard" between `editor` (executable) and `$EDITOR` (variable) as I'm not sure that `$EDITOR` is always defined.

Comment (madumlao) [reply libexec/phpenv-configure]: Suggestion integrated into #87

**Pull request merged**